resources/ansible/mq-automation/roles/mq-install/tasks/mq-create-qm.yaml

```diff
@@ -0,0 +1,20 @@
+- name: Get a list of existing Queue Managers
+  shell: "/opt/mqm/bin/dspmq | awk '{print $1}' | sed 's/^QMNAME(//' | sed 's/)//'"
+  register: qmgrlist
+
+- name: Create Queue Managers
```

Comment:
We can track the loop index and retrieve the corresponding item from the queuemgrs list like so

```yaml
- name: Create Queue Managers
  block:
    - name: Create HA queue manager on standby nodes
      shell: "sudo /opt/mqm/bin/crtmqm -sxs -fs {{ item.size }} {{ item.name }}"
      loop: "{{ queuemgrs }}"
      loop_control:
        index_var: loop_index
      become_user: "mqm"
      when:
        - "'standby' in mq_role"
        - "item.is_ha | bool"
        - "not item.is_dr | bool"
        - "item.name not in qmgrlist.stdout_lines"
``` 

also simplifying when conditions in a list makes this a little cleaner